app/models/shipit/provisioning_handler/base.rb

```diff
@@ -15,6 +15,13 @@ def down
         # Intentionally a noop
       end
 
+      # An (optional) guard to prevent provisioning. Intended to be
+      # use to set logic to determine if enough actual resources exist
+      # to complete the provisioning request.
+      def can_provision?
```

Comment:
```suggestion
      def provision?
```

Does this seem more idiomatic? I _think_ my J2EE roots are showing.

Reply:
yeh, I think the `provision?` seems more rubyst

Reply:
Resolved by: https://github.com/powerhome/shipit-engine/pull/40/commits/aca9e11a1739d8b290f74371fead74c99ea385f3